client/app/components/TypePicker.jsx

```diff
@@ -0,0 +1,85 @@
+import Card from 'antd/lib/card';
+import Input from 'antd/lib/input';
+import List from 'antd/lib/list';
+import { includes, isEmpty } from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Type } from './proptypes';
+import EmptyState from '@/components/items-list/components/EmptyState';
+
+const { Search } = Input;
+const { Meta } = Card;
+
+export default class TypePicker extends React.Component {
+  static propTypes = {
+    types: PropTypes.arrayOf(Type),
+  };
+
+  static defaultProps = {
+    types: [],
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = { searchText: '' };
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  renderListItem(item) {
+    const titleStyle = {
+      fontSize: '13px',
```

Comment:
Maybe, move it somewhere to `.less` file? IMO css-in-js is acceptable when it represent some structural stuff, and if it's unique to specific element; but this one is completely "design" and it's related to a class of elements - no need to set this as inline style for each element - make it a CSS class, and let browser to do it's job.

Reply:
`css-in-js` have always messed up with my concepts :joy:. My parameter so far has been as simple as accepting it for changes that were isolated to the component, but I think I'll adopt yours. Makes total sense for it as a class. I'm thinking of rewriting all styling from here into a `type-picker.less`, however it may get tricky when it comes to handling it over Ant's components.